client/state/sites/selectors/build-seo-title.js

```diff
@@ -1,17 +1,9 @@
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 export default ( titleFormats, type, { site, post = {}, tag = '', date = '' } ) => {
 	const processPiece = ( piece = {}, data ) =>
-		'string' === piece.type ? piece.value : get( data, piece.type, '' );
+		'string' === piece.type ? piece.value : data[ piece.type ] ?? '';
```

Comment:
Any chance for `data` to be `undefined`?

Reply:
`data` is always defined. The `processPiece` and `buildTitle` functions are local and it's easy to verify that all calls include a valid `data` object.